libs/ape-core/ape/core/trainer/community/expel/expel_trainer.py

```diff
@@ -77,6 +77,7 @@ async def train(
                 feedback_history = []
                 prompt_history = []
                 logger.debug(f"Step : {global_step}")
+                print(f"Step : {global_step}")
```

Comment:
We should use `logger.debug` instead of `print` and remove all remaining `print`s